connectors/byoc.py

```diff
@@ -148,7 +148,7 @@ def build_docs_query(self, native_service_types=None, connectors_ids=None):
         return query
 
     def _create_object(self, doc_source):
-        return BYOConnector(
+        return Connector(
```

Comment:
Let's rename the file while we're at it. `byoc.py` and `test_byoc.py` don't have much meaning to an external audience.

Reply:
Totally agree! The reason I didn't rename is because it contains other classes/constants and I want to finish with them first.